lua/lualine/components/branch.lua

```diff
@@ -59,31 +59,29 @@ local function watch_head()
   end
 end
 
--- returns the git_branch value to be shown on statusline
-local function branch()
-  if not git_branch or #git_branch == 0 then return '' end
-  local ok,devicons = pcall(require,'nvim-web-devicons')
-  if ok then
-    local icon = devicons.get_icon('git')
-    if icon ~= nil then
-      return icon .. ' ' .. git_branch
+local function branch(args)
```

Comment:
This is an issue in all components. If it's called `options` in `lualine.lua` then why use `args` here?

Reply:
>This is an issue in all components. If it's called options in lualine.lua

is it ? I've greped the project and couldn't find word options in any code ? This args is the options user gave while declaring component not the M.opt table

Reply:
renamed args to options, M.opt to M.options and settings in utils.lua to options